setup.py

```diff
@@ -10,7 +10,7 @@
     packages=find_packages(),
     zip_safe=False,
     install_requires=[
-        'django>=1.7a,<1.9',
+        'django>=1.10,<1.11',
```

Comment:
I'm not sure what it'd take to be backwards compatible, but I do think it'd be pretty meaningless to say we support (for example) 1.7 unless we at least run the test suite against it regularly.